### PR TITLE
Funny dog mug

### PR DESCRIPTION
**Pull request by ZeroDayDaemon**
## About the PR 
a stylized mug ported from Unitystation. created by @ksivte
**Screenshots**

image

**Changelog**

:cl: Daemon
- add: Added wacky zany funny humorous dog mug reagent container

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
This is clearly the straw that breaks the camel's back. I hope you get repo banned. /s

**Comment by ZeroDayDaemon** [comment]
Up next, porting unity's locker sprites /s

**Comment by Macoron** [comment]
Finally some decent content PRs

**Comment by metalgearsloth** [comment]
> Finally some decent content PRs

https://github.com/unitystation/unitystation/pull/4905

hmm

File: Resources/Textures/Objects/Consumable/Drinks/mug_gromit.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "https://github.com/unitystation/unitystation/blob/develop/UnityProject/Assets/Textures/items/drinks/mug_gromit.asset",
```
Comment (Zumorica) [inline Resources/Textures/Objects/Consumable/Drinks/mug_gromit.rsi/meta.json:8]: > [unitystation/unitystation#4905](https://github.com/unitystation/unitystation/pull/4905)

Given the original PR specifies the spriter, you should include them here too.

**Review by Zumorica — CHANGES_REQUESTED** [review]

**Review by mirrorcult — APPROVED** [review]
funny dog mug

**Pull request merged**